packages/clerk-js/src/ui/components/SignIn/ResetPassword.tsx

```diff
@@ -18,13 +18,16 @@ export const _ResetPassword = () => {
   } = useEnvironment();
   const { failedValidationsText } = usePasswordComplexity(passwordSettings);
 
+  const { t, locale } = useLocalizations();
+
   const passwordField = useFormControl('password', '', {
     type: 'password',
     label: localizationKeys('formFieldLabel__newPassword'),
     isRequired: true,
     enableErrorAfterBlur: true,
     validatePassword: true,
     informationText: failedValidationsText,
+    callback: errors => createPasswordError(errors, { t, locale, passwordSettings }),
```

Comment:
❓ This function accepts an array of one or more errors and returns a string with the final error message. Should we change the naming from `callback` to something less generic?